Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Migration/Tools/MigrationTool.cs

```diff
@@ -0,0 +1,256 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.using System;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+using Object = UnityEngine.Object;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// This tool allows the migration of obsolete components into up-to-date versions.
+    /// In order to be processed by the migration tool, deprecated components require specific implementation of the IMigrationHandler 
+    /// </summary>
+    public class MigrationTool
+    {
+        private List<Type> migrationHandlerTypes = new List<Type>();
+        /// <summary>
+        /// Returns a copy of all loadable implementation types of IMigrationHandler
+        /// </summary>
+        public List<Type> MigrationHandlerTypes { get { return new List<Type>(migrationHandlerTypes); } }
+
+        private List<Object> migrationObjects = new List<Object>();
+        /// <summary>
+        /// Returns a copy of all Game Objects, Prefabs and Scene assets selected for migration
```

Comment:
```suggestion
        /// Returns a copy of all GameObjects, Prefabs and Scene assets selected for migration
```

Reply:
(either use class name or if you want to just write game object use lower case :)  - applying doc styleguide as these comments show up in the API docs )